internal/vlan/retrieve.go

```diff
@@ -32,14 +32,14 @@ func (c *Client) Retrieve() *cobra.Command {
 
 	// retrieveVirtualNetworksCmd represents the retrieveVirtualNetworks command
 	var retrieveVirtualNetworksCmd = &cobra.Command{
-		Use:     "get",
+		Use: `get -p <project_UUID>`,
 		Aliases: []string{"list"},
-		Short:   "Retrieves a list of virtual networks for a single project.",
-		Long: `Example:
-
-metal virtual-network get -p [project_UUID]
-
-	`,
+		Short: "Lists virtual networks.",
+		Long: "Retrieves a list of all VLANs for the specified project.",
+		Example:  `
+  # Lists virtual networks for project 3b0795ba-ec9a-4a9e-83a7-043e7e11407c:
+  virtual-network get -p 3b0795ba-ec9a-4a9e-83a7-043e7e11407c`,
```

Comment:
Should this say "metal virtual-network get -p 3b0795ba-ec9a-4a9e-83a7-043e7e11407c"?

Reply:
yes, good catch.